### PR TITLE
Updated the Readme

### PR DESCRIPTION
**Pull request by ShaileshKumar007**
Beautify the documentation. Have changed some lines to Pascalcase letters and have given the indentation in subparts of the headings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,38 +14,38 @@ YouTube link https://www.youtube.com/watch?v=0lOAOQaTNdw
...
 
 3a. Navigate into the directory `cd api-datastax`
 
 3b. Initialise npm with `npm init -y`
 
-### 4. testing
+### 4. Testing:
 
-4a. Install PactumJS dependency `npm install pactum`
+&nbsp; 4a. Install PactumJS dependency `npm install pactum`
```
Comment (eddiejaoude) [inline README.md:33]: Not sure why this is required?

File: README.md [hunk 2]
```diff
@@ -87,16 +87,19 @@ Failures:
...
-6a. sign up http://links.eddiejaoude.io/t/BQe90ur6AYCp
-6b. create database `api`
-6c. create keyspace `apidemo`
-6d. create token
-
-6e. set environment variables (treat these as passwords)
-
-6f. insert data using curl
+&nbsp; 6a. Sign up http://links.eddiejaoude.io/t/BQe90ur6AYCp
+</br> </br>
```
Comment (eddiejaoude) [inline README.md:93]: Not sure why these extra line breaks are needed?

**Comment by eddiejaoude** [comment]
Thank you for the contribution, but I think the formatting with extra html is not needed

**Comment by ShaileshKumar007** [comment]
Actually, that's my first contribution so I thought I should start with formatting. I will take care of these things next time.
Thank you for your guidance.

**Pull request closed without merging**